Classifiers/classifier-Crowdstrike-Streaming.json

```diff
@@ -0,0 +1,56 @@
+{
+	"id": "5d9a5ec5-33cb-4f82-8c5e-e8ae6f7831d8",
+	"version": -1,
+	"modified": "2018-05-21T12:41:29.542577629Z",
+	"defaultIncidentType": "Malware",
+	"mapping": {
+		"Malware": {
+			"internalMapping": {
+				"Dest": {
+					"simple": "event.ComputerName",
+					"complex": null
+				},
+				"Dest NT Domain": {
+					"simple": "event.MachineDomain",
+					"complex": null
+				},
+				"File Hash": {
+					"simple": "event.MD5String",
+					"complex": null
+				},
+				"File Name": {
+					"simple": "event.FileName",
+					"complex": null
+				},
+				"File Path": {
+					"simple": "event.FilePath",
+					"complex": null
+				},
+				"User": {
+					"simple": "event.UserName",
+					"complex": null
+				},
+				"Vendor Product": {
+					"simple": "\"CrowdStrike\"",
+					"complex": null
+				},
+				"details": {
+					"simple": "event.DetectDescription",
+					"complex": null
+				}
+			},
+			"dontMapEventToLabels": false
+		}
+	},
+	"unclassifiedCases": {},
+	"custom": true,
+	"transformer": {
+		"simple": "metadata.eventType",
```

Comment:
if all events should map to "Malware" - we can (and perhaps should) remove the transformer. `defaultIncidentType` is sufficient (we need to improve the UX on that manner)